js/web/lib/wasm/jsep/webgpu/ops/fuse-utils.ts

```diff
@@ -11,7 +11,7 @@ export interface InternalActivationAttributes {
 }
 
 export const getActivationSnippet = (attributes: InternalActivationAttributes, valueType: string):
-    {activationFunction: string; applyActivation: string;} => {
+    {activationFunction: string; applyActivation: string} => {
```

Comment:
```suggestion
    {activationFunction: string, applyActivation: string} => {
```

Reply:
Changing it to comma will cause the lint error:
```
/workspace/users/jiajia/onnxruntime/js/web/lib/wasm/jsep/webgpu/ops/fuse-utils.ts
  14:33  error  Expected a semicolon  @typescript-eslint/member-delimiter-style

Reply:
Both should be OK, but our .eslintrc.js has below configuration, so comma is not allowed here.
    '@typescript-eslint/member-delimiter-style': [
      'error', {
        'multiline': { 'delimiter': 'semi', 'requireLast': true },
        'singleline': { 'delimiter': 'semi', 'requireLast': false }
      }
    ],

Reply:
resolving this one so we can merge the fix